app/src/routes/(public)/inscription/+page.svelte

```diff
@@ -55,8 +55,8 @@
 			<h1>Demande d'inscription envoyée</h1>
 			<p>Nous avons bien pris en compte votre demande de nouvelle inscription.</p>
 			<p>
-				Vous recevrez un courriel de confirmation, avec un lien pour vous connecter au Carnet de
-				bord.
+				Si votre inscription est validée, vous recevrez un courriel de confirmation, avec un lien
+				pour vous connecter au Carnet de bord.
```

Comment:
Pour la cohérence je propose :
```suggestion
				pour vous connecter à Carnet de bord.
```

Reply:
Haha bonne remarque, je me suis posé la question aussi. Je suis sûr que ce n’est pas le seul endroit incohérent, on a aussi `Bord` vs `bord` parfois… mais je prends celle-ci !